src/_data/sites/cambopedia.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://cambopedia.com/",
+    "name": "Cambopedia",
+    "description": "Local insights into living in Cambodia",
+    "twitter": "https://twitter.com/cambopedia",
+    "authoredBy": ["cambopedia"],
+    "source_url": "https://cambopedia.com/"
```

Comment:
@tharumbun This is an optional field, but should point to the source code (in GitHub or wherever), if possible. 
If this isn't an open-source type repo/project, we can delete this property.

* `url`: The site’s production URL
* `name`: Name of the site
* `description`: A short text description of the site
* `twitter`: Twitter username for the site or the site’s author.
* `authoredBy`: An array of Twitter usernames of the site’s authors. Supplements the `twitter` entry. (Optional)
* `source_url`: URL to the source code (Optional)

Reply:
@tharumbun Did you still want to merge this?
I don't seem to be able to edit the files anymore [to remove the `source_url`] since it looks like you deleted your fork of the repository.

Reply:
Okay. Thanks

Reply:
@tharumbun You'll either need to update the `source_url` to a valid source code URL (if the site is open source), or else remove that field. I don't seem to be able to modify the files anymore via GitHub UI.